app/models/sealion.rb

```diff
@@ -0,0 +1,5 @@
+class Sealion < ActiveRecord::Base
+  # Remember to create a migration!
+  validates :first_name, {:presence => true}
```

Comment:
The tradeoffs I'm thinking of are: doing a validation in the model guarantees that we'll never write bad data, but it doesn't guarantee that bad data doesn't exist in the database, since there are many ways of writing to the database. On the other hand, doing validation in the database means that we need to trust the database to  actually do that validation, which probably works, but depends on the database.